pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -141,6 +141,10 @@ func (s *Server) initK8SConfigStore(args *PilotArgs) error {
 			return err
 		}
 	}
+	s.RWConfigStore, err = configaggregate.MakeWriteableCache(s.ConfigStores, configController)
```

Comment:
Do we need a RWConfigStore and a s.configController? I was expecting just a single s.configController, which happened to be RW

Reply:
`s.configController` needs to be initialized even when k8s is not being used, and I don't see how to accommodate that while also setting it to a WriteableCache using the k8s configstore.  I'm not saying it's not possible, but the bootstrap classes are rats' nests of order dependant operations, and I'm not confident I could make that change without side effects.